spec/system/schools/advice_pages/gas_out_of_hours_spec.rb

```diff
@@ -3,14 +3,16 @@
 RSpec.describe "gas out of hours advice page", type: :system do
 
   let(:key) { 'gas_out_of_hours' }
-  let(:expected_page_title) { "Gas out of hours usage analysis" }
+  let(:expected_page_title) { "Out of school hours gas use" }
   include_context "gas advice page"
 
   context 'as school admin' do
 
     let(:user)  { create(:school_admin, school: school) }
 
     before do
+      allow_any_instance_of(Usage::AnnualUsageBreakdownService).to receive(:usage_breakdown) { nil }
```

Comment:
It would be better if you could include some basic checks for each page (applies to the electricity one too), e.g:

- are the right sections showing?
- has the right chart(s) been added
- are the expected numbers on the page

The latter means stubbing the data, but should all be fairly simple.